src/hexagon.core.js

```diff
@@ -149,6 +149,7 @@ export { inputGroup } from 'components/input-group';
 export { group, section } from 'components/layout';
 export { dragContainer, DragContainer } from 'components/drag-container';
 export { progressBar, ProgressBar } from 'components/progress-bar';
+export { visualisationBar, VisualisationBar } from 'components/visualisation-bar';
```

Comment:
The tests will fail due to these exports. Need to add them to `hexagon.spec.js` (there's a big list of exported members in the `hx export the right things` test)